docs/source/upcoming_release_notes/1361-IntegralSigned-Addition-AdcSample-BracketRemoval.rst

```diff
@@ -0,0 +1,30 @@
+1361 IntegralSigned-Addition-AdcSample-BracketRemoval
+#################
+
+API Breaks
+----------
+- Changing PVs for sample components in Wave8V2ADCSamples by removing brackets
```

Comment:
You might even consider this to not be an api break from the POV of a user of the python, since we're preserving the component names.  If anything we're conforming to the changed api of the backing EPICS IOC